plugin/keyring_vault/keyring_vault-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+ADD_EXECUTABLE(keyring_vault-test
```

Comment:
Please see what is done with e.g. storage/heap/hp_test1 - an arbitrary executable picked up by "make test"